observations/obs_converters/AIRS/work/mkmf_L1_AMSUA_to_netcdf

```diff
@@ -0,0 +1,29 @@
+#!/bin/csh
+#
+# DART software - Copyright UCAR. This open source software is provided
+# by UCAR, "as is", without charge, subject to all terms of use at
+# http://www.image.ucar.edu/DAReS/DART/DART_download
+
+if ( $?NCAR_INC_HDFEOS ) then
+   set HDFEOS_INCLUDE = $NCAR_INC_HDFEOS
+   set HDFEOS_LIBROOT = $NCAR_LDFLAGS_HDFEOS
+   set HDFEOS_LIBRARIES = $NCAR_LIBS_HDFEOS
+
+else if ( -e /glade/work/thoar/hdf-eos/lib/libhdfeos.a ) then
+   set HDFEOS_INCLUDE = "/glade/work/thoar/hdf-eos/include"
+   set HDFEOS_LIBROOT = "/glade/work/thoar/hdf-eos/lib"
+   set HDFEOS_LIBRARIES = "-lhdfeos -lGctp -lmfhdf -ldf -ljpeg -lz"
+
+else
+   set HDFEOS_INCLUDE = "/Users/thoar/gnu/gnu_9.2.0/hdf-eos/include"
+   set HDFEOS_LIBROOT = "/Users/thoar/gnu/gnu_9.2.0/hdf-eos/lib"
+   set HDFEOS_LIBRARIES = "-lhdfeos -lGctp -lmfhdf -ldf -ljpeg -lz"
+endif
```

Comment:
Again ... why this works interactively but not when run as a batch job is beyond me.
Note the subtlety in that _there is no hdf-eos module_ on cheyenne, so the first 'if' block is moot.

Reply:
are these setting internal make variables, shell variables (which kind - tcsh or ksh), or environment variables?  what gets preserved might be different depending on interactive vs batch system, maybe?

Reply:
So ... what is happenning is that the build works fine - but the LD_LIBRARY_PATH is not necessarily set consistent with these. I made notes in the affected mkmf files (see lines 7-10 here) and in the `run_tests.csh`  script.